### DISCUSSION TITLE
Inverse of extract_image_chip()

### DISCUSSION THREAD

**Discussion by pfeatherstone** [davisking/dlib#2857] [Q&A] [answered]
Is there a way to get an inverse transformation for `extract_image_chip()` ? Looking at the code, it's a bit more complication than I originally thought. However, shouldn't it just be a `point_affine_tranforsmation` (rotation + translation) + resize ? In which case, those should all be invertible with resize  + `inv(point_affine_tranforsmation(...))` ?

I'm trying to remap the landmark detections from a custom model computed in the extract chip, back into the original image.

**Comment by arrufat** [comment, marked as answer]
Does https://github.com/davisking/dlib/pull/2781 help?

**Comment by pfeatherstone** [comment]
So use `get_mapping_to_chip()` then possible `inv()` ? I'll check it out.
